CMakeLists.txt

```diff
@@ -92,6 +93,9 @@ if (Qt5Core_FOUND AND NOT BUILD_WITH_QT6)
   include_directories(${Qt5Core_INCLUDE_DIRS})
 else()
   find_package(Qt6 COMPONENTS Core REQUIRED)
+  if(QTKEYCHAIN_BUILD_WITH_QML)
+      find_package(Qt6 COMPONENTS QML REQUIRED)
```

Comment:
I'm a bit unsure how this dependency fits in with distro packages and the like, one can either disable the dependency to not have the qtkeychain package depend on QML, or enable it, and add the dependency. I guess it's maximally annoying for application developers if then some distros enable QML support and some ignore the flag and thus don't.
I'm leaning to enable QML support by default, and let those who want a minimal build do custom builds with the flag set to OFF.

Reply:
As a developer of an offscreen library depending on Qt Keychain I'd rather be happy not to drag the dependency on QML. As for the CMake part, I think there's a way to introduce this dependency down the line in `*Config.cmake` file instead and that can even be conditional, but I didn't try it.

Reply:
Are you relying on distro packages, or are pulling in qtkeychain directly?

Reply:
In CI, I pull Qt Keychain from a tag and build it directly because it's the only way to have one unified logic across 3 platforms; but distros packaging my library have all reasons to link it externally.